wpimath/src/main/native/include/frc/geometry/Translation3d.h

```diff
@@ -148,6 +148,18 @@ class WPILIB_DLLEXPORT Translation3d {
                          units::meter_t{qprime.Z()}};
   }
 
+  /**
+   * Rotates this translation around another translation in 3D space.
+   *
+   * @param other The other translation to rotate around.
+   * @param rot The rotation to rotate the translation by.
+   * @return The new rotated translation.
+   */
+  constexpr Translation3d RotateAround(const Translation3d& other,
+                                       const Rotation3d& rot) const {
+    return operator-(other).RotateBy(rot).operator+(other);
```

Comment:
```suggestion
    return (*this - other).RotateBy(rot) + other;
```
The `operator-` and `operator+` names tell C++ to use them when using the regular arithmetic operators `-` and `+`, so we might as well use them to make the code more readable.